run/action.yml

```diff
@@ -0,0 +1,41 @@
+name: 'run-cmd'
+description: 'Run command'
+inputs:
+  branch:  # id of input
+    description: 'Branch name'
+    required: true
+  command:
+    description: "Command to be run"
+    required: true
+runs:
+  using: "composite"
+  steps:
+      - name: Download repo
+        uses: actions/checkout@v4.1.1
+        with:
+          ref: ${{ inputs.branch }}
+      - name: Generate script
+        shell: bash
+        # Here it would get the script from previous step
+        run: |
+          echo "$COMMAND" >> command.sh
+          chmod +x command.sh
+          ./command.sh
+          rm command.sh
+        env:
+          COMMAND: ${{ inputs.command }}
+      - name: Report failure
+        if: ${{ failure() }}
+        shell: bash
+        run: echo "Command failed!"
+      - name: Check changes
+        shell: bash
+        run: |
+          git config --global --add safe.directory /__w/${{ github.event.repository.name }}/${{ github.event.repository.name }}
```

Comment:
Would it be cleaner to use `.` or `$(pwd)`?

Reply:
That‘s a good idea, I‘ll try that!

Reply:
I tried but end up reverting it as the running directory is `/home/runner/work/polkadot-sdk/polkadot-sdk`

Reply:
Do you have a log of a failed run because of this? 🤔

Reply:
Here is tge test run: https://github.com/paritytech-stg/polkadot-sdk/actions/runs/8927469091/job/24520953847

Reply:
And do you have a failed run log when you dont configure the safe directory at all?

Reply:
I decided to replace it with [stefanzweifel/git-auto-commit-action](https://github.com/stefanzweifel/git-auto-commit-action).